client/us_census_variable_names.py

```diff
@@ -0,0 +1,2 @@
+TOTAL_HOUSEHOLDS_RECEIVING_FOOD_STAMPS_IN_LAST_YEAR = "B22001_002E"
```

Comment:
Is this supposed to be hardcoded?

Reply:
Yeah "B22001_002E" is the name of this variable in the US Census api.